stripe/api_resources/customer.py

```diff
@@ -43,6 +44,12 @@ class Customer(
 ):
     OBJECT_NAME = "customer"
 
+    def create_funding_instructions(self, idempotency_key=None, **params):
+        url = self.instance_url() + "/funding_instructions"
+        headers = util.populate_headers(idempotency_key)
+        self.refresh_from(self.request("post", url, params, headers))
```

Comment:
This shouldn't refresh_from and instead return the funding instructions

Reply:
Fixed!